config/environments/production.rb

```diff
@@ -73,10 +73,10 @@
 
   config.action_mailer.smtp_settings = {
    :port =>           '587',
-   :address =>        'smtp.sendgrid.net',
-   :user_name =>      ENV['SENDGRID_USERNAME'],
-   :password =>       ENV['SENDGRID_PASSWORD'],
-   :domain =>         'heroku.com',
+   :address =>        'email-smtp.us-east-1.amazonaws.com',
+   :user_name =>      ENV['SMTP_USERNAME'],
+   :password =>       ENV['SMTP_PASSWORD'],
+   :domain =>         'mylibrarynyc.org',
```

Comment:
Is there email setting for local/dev/qa?

Reply:
Technically, all environments email settings are being called in  config/initializers/setup_mail.rb . I can remove this code.